src/timezones/timezoneManager.js

```diff
@@ -60,10 +60,9 @@ export class TimezoneManager {
 		}
 
 		if (this._aliases.has(timezoneId)) {
-			const realTimezoneId = this._aliases.get(timezoneId)
-			if (this._timezones.has(realTimezoneId)) {
-				return this._timezones.get(realTimezoneId)
-			}
+			const resolvedTimezoneId = this._aliases.get(timezoneId)
+			// can be a recursive alias:
+			return this.getTimezoneForId(resolvedTimezoneId)
```

Comment:
Should we limit the recursion to avoid infinite loops in case the zone information is bad?